### PR TITLE
BREAKING: KBA - Remodels the fallback language mechanism for the knowledgebase system in order to support multiple fallback languages

### PR DESCRIPTION
**Pull request by mko-sci**
Fixes: SIRI-865

### CODE REVIEW

**Review by idlira — APPROVED** [review]
Probably someone would prefer .stream() versus regular loops :-P

**Review by andyHa — APPROVED** [review]

File: src/main/java/sirius/biz/tycho/kb/KnowledgeBase.java [hunk 1]
```diff
@@ -125,7 +129,15 @@ public final String getFallbackLang() {
      * @return the default language as two-letter ISO code
      */
     public String getFallbackLanguage() {
-        return fallbackLanguage;
+        return !fallbackLanguages.isEmpty() ? null : fallbackLanguages.get(0);
+    }
```
Comment (andyHa) [inline src/main/java/sirius/biz/tycho/kb/KnowledgeBase.java:133]: Check if we can kill this method and the getFallbackLang above

Comment (mko-sci) [reply src/main/java/sirius/biz/tycho/kb/KnowledgeBase.java]: we can kill it since they are not used among scireum-core and ox =)

**Comment by mko-sci** [comment]
> Probably someone would prefer .stream() versus regular loops :-P

But its way easier to maintain and understand since there is already a lot of stuff going on inside the loop

**Review by fhaScireum — APPROVED** [review]

**Review by idlira — APPROVED** [review]

**Pull request merged**